includes/database/NotificationsDB.php

```diff
@@ -0,0 +1,251 @@
+<?php
+/**
+ * Notifications Database
+ *
+ * @package   easy-digital-downloads
+ * @copyright Copyright (c) 2021, Easy Digital Downloads
+ * @license   GPL2+
+ * @since     2.11.4
+ */
+
+namespace EDD\Database;
+
+use EDD\Models\Notification;
+use EDD\Utils\EnvironmentChecker;
+use EDD\Utils\NotificationImporter;
+
+class NotificationsDB extends \EDD_DB {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		global $wpdb;
+
+		$this->table_name  = $wpdb->prefix . 'edd_notifications';
+		$this->primary_key = 'id';
+		$this->version     = '0.2'; // @todo update to 1.0 pre-release
```

Comment:
Will this be updated as part of merging this PR or in the final release branch?

Reply:
I'll update it to `1.0` as part of this PR. Just want to do that once we're confident I won't be making anymore DB changes in this PR.